composer.lock

```diff
@@ -2124,6 +2124,83 @@
             },
```

Comment:
@greggmarshall could you verify that this modification looks of to you? I had a merge conflict with the hashes when switching to my branch which required stashing, and I had to manually pick one of the two hash value options.